image_editor_platform_interface/lib/src/option/color.dart

```diff
@@ -117,6 +117,15 @@ class ColorOption implements Option {
 
   factory ColorOption.contrast(double contrast) {
     final m = List<double>.from(defaultColorMatrix);
+
+    /// The contrast translate follows the guide at:
+    /// https://docs.rainmeter.net/tips/colormatrix-guide
```

Comment:
Can you place this before the factory constructor?
```dart
  /// The contrast translate follows the guide:
  /// https://docs.rainmeter.net/tips/colormatrix-guide.
  factory ColorOption.contrast(double contrast) {
```

Reply:
done.

Reply:
any other modification needed?

Reply:
Yes, please consider adding a unit test against this.

Reply:
Actually its very simple math and to test it is like again writing the same matrix. Since I could not any tests folder in this package, I did not push. Here's a unit test:

```
import 'package:image_editor_platform_interface/src/option/edit_options.dart';
import 'package:test/test.dart';

void main() {
  test('contrast = 1 should give us the default matrix', () {
    expect(ColorOption.contrast(1).matrix, defaultColorMatrix);
  });
  test(
      'contrast = 0 should have zeros in the diameter, 127.5 in last column'
      'except for the last row', () {
    final matrix = List.filled(defaultColorMatrix.length, 0.0);
    matrix[4] = 127.5;
    matrix[9] = 127.5;
    matrix[14] = 127.5;
    matrix[18] = 1;
    expect(ColorOption.contrast(0).matrix, matrix);
  });
}
```

Reply:
Anything more?! The current contrast matrix is obviously wrong and it's just brightness matrix.

I'm still waiting for this commit to be accepted so that I can use this package from its main repository instead of my local one.